identity/bootstrapper.go

```diff
@@ -35,25 +36,23 @@ func (*Bootstrapper) Bootstrap(context map[string]interface{}) error {
 		return err
 	}
 
-	IDService = NewEthereumIdentityService(cfg, idFactory, registryContract, ethereum.GetClient,
+	if _, ok := context[bootstrap.BootstrappedQueueServer]; !ok {
+		return errors.New("queue hasn't been initialized")
+	}
+	queueSrv := context[bootstrap.BootstrappedQueueServer].(*queue.Server)
+
+	IDService = NewEthereumIdentityService(cfg, idFactory, registryContract, queueSrv, ethereum.GetClient,
```

Comment:
I thought we got rid of this `IDService`, didn't we?

Reply:
thats the other pr